src/libs/notifications/notifications.js

```diff
@@ -0,0 +1,28 @@
+import { navigateToApp } from '/libs/functions/openApp'
+import { generateWebLink } from 'cozy-client'
+
+const navigateFromNotification = (client, navigate, notification) => {
+  const { data: { url, slug } = {} } = notification
+
+  if (!url || !slug) return
+
+  const subDomainType = client.getInstanceOptions().capabilities.flat_subdomains
+    ? 'flat'
+    : 'nested'
+
+  const href = generateWebLink({
+    cozyUrl: client.getStackClient().uri,
+    subDomainType,
+    pathname: '/',
```

Comment:
I'm wondering if the pathname should not be part of the notification's content. It can be /public, /export or else. It should not be hardcoded.